email_marketing/mailing_lists.rst

```diff
@@ -0,0 +1,53 @@
+=======================
+Work with Mailing Lists
+=======================
+Mailing lists could be a goldmine for your company’s marketing department as they can provide leads
+for sales, focus groups for testing new products, and consumers to provide feedback on satisfaction,
+for example.
+
+Create Mailing lists
+=====================
+Go to :menuselection:`Mailing lists --> Mailing lists --> Create`.
+Enable the option *Is Public* to make the list visible for users when unsubscribing.
+
+
+.. image:: media/mailing1.png
+   :align: center
+   :alt: Mailing lists in Odoo Email Marketing
+
+Once your mailing list is created, you can manually add contacts clicking on *Create*.
+*Import* a list of contacts if you have it from a previous tool.
```

Comment:
Also, it may make perfect sense to emphasize both terms but the final result looks weird in my opinion:
"... *Create*. *Import* ..."